beta/google-cloud-run/README.md

```diff
@@ -0,0 +1,186 @@
+
+> [!NOTE]
+> If you don't have a Google Cloud account, you can sign up for a free trial with starting credit: <https://console.cloud.google.com/freetrial>
+
+## Step 1: Set up Google Cloud
+
+1. Sign in to the Google Cloud console and activate Cloud Shell: <https://console.cloud.google.com?cloudshell=true>
+2. Create a [project](https://cloud.google.com/docs/overview#projects) to organize the Google Cloud resources for your 1Password SCIM Bridge deployment, and set it as the default project for your Cloud Shell environment:
+
+    ```sh
+    gcloud projects create op-scim-bridge --set-as-default
+    ```
+
+> [!TIP]
+> If you have already created a project for SCIM Bridge, set it as the default project for this Cloud Shell session. For example:
+>
+> ```sh
+> gcloud config set project op-scim-bridge
+> ```
+
+3. Enable the Secret Manager and Cloud Run APIs for your project:
+
+    ```sh
+    gcloud services enable secretmanager.googleapis.com run.googleapis.com
+    ```
+
+4. Set the default region for Cloud Run:
+
+    ```sh
+    gcloud config set run/region us-central1
+    ```
+
+> [!NOTE]
+> All region-bound resources created in the following steps will be created in the specified region. You may replace `us-central1` in the above commmand with your preferred region.
+
+## Step 2: Create a secret for your `scimsession` credentials
+
+The Cloud Run service for SCIM Bridge will be configured to mount volume using a secret from Secret Manager. Follow these steps to upload your `scimsession` credentials file to the Cloud Shell, create a secret, and store the file contents as its first secret version:
+
+1. Click **⋮** _(More)_ > **Upload** in the Cloud Shell terminal menu bar.
+2. Click **Choose Files**. Select the `scimsession` file that you saved to your computer.
+3. Use the suggested destination directory. Click **Upload**.
+> [!NOTE]
+> If the file is saved to a different directory or using a different file name, make a note of the full path to
+> the file.
+4. Create a secret with the contents of this file as its first secret version using the following command:
+
+    ```sh
+    gcloud secrets create scimsession --data-file=$HOME/scimsession
+    ```
+
+> [!TIP]
+> The command above is expected work as is if the file is named `scimsession` and if it was saved to the home
+> directory when uploading the file. If not, replace `$HOME/scimsession` with the actual path to the file. For
+> example:
+>
+> ```sh
+> gcloud secrets create scimsession --data-file=/example/path/to/scimsession.file
+> ```
+
+5. Enable Cloud Run to access the secret using the Compute Engine default service account for the project:
+
+    ```sh
+    gcloud secrets add-iam-policy-binding scimsession --member=serviceAccount:$(
+      gcloud iam service-accounts list --filter="$(
+        gcloud projects describe op-scim-bridge --format="value(projectNumber)"
```

Comment:
We might want to add a tip here, stating that if they used a custom project name to edit this command. 

As an alternative to a tip, we could set a `opProject` variable as part of the command used in step 1.2 and the tip in step 1.2 for customers who already created a project. This way we could simply use that within this long command to remove any confusion.

Reply:
Great point, and I think we can likely improve the flow a bit in that earlier step _and_ perhaps replace this entire line in this block with that variable.

Let's merge this as is and then I'll note it in a new issue.